TurkishFilter/sections/antiadblock.txt

```diff
@@ -11,7 +11,11 @@
 ankarakampkafasi.com,zamaninvarken.com,kriptoradar.com,bakimlikadin.net,korsanedebiyat.com,genelpara.com,azbuz.org,mustafabukulmez.com#%#//scriptlet("prevent-setTimeout", "0===o.offsetLeft&&0===o.offsetTop")
 teknop.net,ozgunbilgi.com#%#//scriptlet("set-constant", "adblock.check", "noopFunc")
 !
-!
+! https://github.com/AdguardTeam/AdguardFilters/issues/221122
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=adrbook.com
+!#if (adguard_app_ios || adguard_ext_android_cb || adguard_ext_safari)
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,domain=adrbook.com
+!#endif
```

Comment:
The directive is not needed. You can use the hint instead of the rules or these ones below.
```suggestion
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=adrbook.com,important
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,domain=adrbook.com
```